instance/config.py

```diff
@@ -4,8 +4,8 @@ class Config(object):
     #parent configuration class
     DEBUG = False
     CSRF_ENABLED = True
-    SQLALCHEMY_DATABASE_URI = os.getenv('DATABASE_URL')
-    SECRET = os.getenv('SECRET')
+    SQLALCHEMY_DATABASE_URI = 'postgresql://localhost/brighter_events'
+    SECRET_KEY = "super awesome secret #$%^*("
```

Comment:
It's good practice to hide `SECRET_KEY` and `SQLALCHEMY_DATABASE_URI` and export it from local .env file

Reply:
Should I have the file in root root directory of the project? Should it be included in gitignore?

Reply:
Yes it should be in the root directory of the project, yes should include it in .gitignore, it contains sensitive info, for security reasons keep it our of GitHub.
Then run the project from that file